src/action/bandaidset.rs

```diff
@@ -133,7 +133,7 @@ impl FirstAidKit {
                 } else {
                     // Original content is shorter than replacement
                     let insertion = LineColumn {
-                        line: span.end.line,
+                        line: span.end.line + 1,
```

Comment:
No `+1` without an explicit explanation why this is needed. That cause a shit ton of refactoring before :) and I don't feel a need to repeat that mistake.

Reply:
Yeah I see that :smile: I try to avoid it in the first place, but I don't think its possible :thinking: